robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -120,6 +126,20 @@ public InstrumentationConfiguration build() {
           ShadowedObject.class,
           TempDirectory.class
       ));
+      packagesToNotAquire.addAll(Arrays.asList(
+          "java.",
+          "javax.",
+          "sun.",
+          "com.sun.",
+          "org.w3c.",
+          "org.xml.",
+          "org.junit",
+          "org.hamcrest",
+          "org.specs2",  // allows for android projects with mixed scala\java tests to be
+          "scala.",      //  run with Maven Surefire (see the RoboSpecs project on github)
+          "kotlin.",
+          "com.almworks.sqlite4java" // Fix #958: SQLite native library must be loaded once.
```

Comment:
This should really have a trailing dot right? Otherwise `com.almworks.sqlite4javaClass` will not be acquired even though its not in the `com.almworks.sqlite4java` package.